po/fr.po

```diff
@@ -3251,699 +3258,723 @@ msgstr "Mauvaise expression régulière pour le type de fichier %s : %s"
 msgid "Unfold all"
 msgstr "Tout déplier"
 
-#: ../src/keybindings.c:676
+#: ../src/keybindings.c:679
 msgid "Reload symbol list"
 msgstr "Recharger la liste des symboles"
 
-#: ../src/keybindings.c:678
+#: ../src/keybindings.c:681
 msgid "Remove Markers"
 msgstr "Supprimer les marqueurs"
 
-#: ../src/keybindings.c:680
+#: ../src/keybindings.c:683
 msgid "Remove Error Indicators"
 msgstr "Supprimer les indicateurs d'erreurs"
 
-#: ../src/keybindings.c:682
+#: ../src/keybindings.c:685
 msgid "Remove Markers and Error Indicators"
 msgstr "Supprimer les marqueurs et les indicateurs d'erreurs"
 
-#: ../src/keybindings.c:687 ../src/toolbar.c:72
+#: ../src/keybindings.c:690 ../src/toolbar.c:72
 msgid "Compile"
 msgstr "Compiler"
 
-#: ../src/keybindings.c:691
+#: ../src/keybindings.c:694
 msgid "Make all"
 msgstr "Make all"
 
-#: ../src/keybindings.c:694
+#: ../src/keybindings.c:697
 msgid "Make custom target"
 msgstr "Make custom target"
 
-#: ../src/keybindings.c:696
+#: ../src/keybindings.c:699
 msgid "Make object"
 msgstr "Make object"
 
-#: ../src/keybindings.c:698
+#: ../src/keybindings.c:701
 msgid "Next error"
 msgstr "Erreur suivante"
 
-#: ../src/keybindings.c:700
+#: ../src/keybindings.c:703
 msgid "Previous error"
 msgstr "Erreur précédente"
 
-#: ../src/keybindings.c:702
+#: ../src/keybindings.c:705
 msgid "Run"
 msgstr "Exécuter"
 
-#: ../src/keybindings.c:704
+#: ../src/keybindings.c:707
 msgid "Build options"
 msgstr "Options de construction"
 
-#: ../src/keybindings.c:709
+#: ../src/keybindings.c:712
 msgid "Show Color Chooser"
 msgstr "Afficher le sélecteur de couleurs"
 
-#: ../src/keybindings.c:982
+#: ../src/keybindings.c:995
 msgid "Keyboard Shortcuts"
 msgstr "Raccourcis clavier"
 
-#: ../src/keybindings.c:994
+#: ../src/keybindings.c:1007
 msgid "The following keyboard shortcuts are configurable:"
 msgstr "Les raccourcis clavier suivants peuvent être configurés :"
 
-#: ../src/keyfile.c:1027
+#: ../src/keyfile.c:1033
 msgid "Type here what you want, use it as a notice/scratch board"
 msgstr ""
 "Écrivez ici ce que vous souhaitez, utilisez cela comme un tableau de notes"
 
-#: ../src/keyfile.c:1254
+#: ../src/keyfile.c:1260
 msgid "Failed to load one or more session files."
 msgstr "Impossible de charger un ou plusieurs fichiers de session."
 
-#: ../src/libmain.c:118
+#: ../src/libmain.c:122
 msgid ""
-"Set initial column number for the first opened file (useful in conjunction "
-"with --line)"
+"Set initial column number to COLUMN for the first opened file (useful in "
+"conjunction with --line)"
 msgstr ""
-"Définit la colonne initiale pour le premier fichier ouvert (utile en "
-"association avec --line)"
+"Définit COLONNE comme colonne initiale pour le premier fichier ouvert (utile "
+"en association avec --line)"
+
+#: ../src/libmain.c:122
+msgid "COLUMN"
+msgstr "COLONNE"
+
+#: ../src/libmain.c:123
+msgid "Use alternate configuration directory DIR"
+msgstr "Utilise DIR comme dossier de configuration alternatif"
```

Comment:
As you translated `DIR` to `DOSSIER` (which makes sense), you should use `DOSSIER` here as well

Reply:
C'est commun a traduire les balise comme ca?

Reply:
DOS? :grin:

Reply:
I'm Anglo, but I think "dossier" means "folder", like "the NSA has a dossier on you" :)

Reply:
@codebrainz AFAIK it's common to translate it, and GTK does (`--display=AFFICHAGE` in Geany's options).

@elextr well… technically why not but I don't like the sound of it for some reason 😁 

@codebrainz yeah, but we don't really have 2 words here and it's the common wording in French for a file system "directory" (or "folder", as it's sometimes worded, at least on some icons and stuff)  -- not even sure what the difference is in English.

Reply:
@b4n yeah, I misunderstood @elextr's comment as (sarcastically) asking what is a dossier rather suggesting an abbreviation. FWIW, in the context of computers in English "folder" and "directory" are identical, but otherwise a directory is like an index/listing (ex. phone book), I think the French word is "annuaire".

Reply:
@codebrainz yeah @b4n was right, I was suggesting the same abbreviation as with directory, the first three letters, then suggesting it might perhaps not be suitable, sad that a perfectly good three letter combination is forever ruined :)

Reply:
I chose not to use an abbreviation here since we have enough room (the english version uses FILE/COLUMN/DIR)